Dockerfile

```diff
@@ -5,6 +5,10 @@ LABEL maintainer="SDF Ops Team <ops@stellar.org>"
 RUN mkdir -p /app
 WORKDIR /app
 
+ARG REACT_APP_SENTRY_KEY
```

Comment:
This will cause the sentry key to be built into the resulting image, even if it doesn't end up in the image filesystem it'll be in the image meta data.

Reply:
I'm not sure what's the best way to serve these envs either...

Reply:
Will be addressed in: https://app.zenhub.com/workspaces/product-engineering-5f7cd2027d859300120adb76/issues/stellar/product-engineering/260